keylime/cmd/create_policy.py

```diff
@@ -151,17 +157,11 @@ def process_ima_buf_in_measurement_list(
                     if path in ignored_keyrings or not get_keyrings:
                         continue
 
-                    if path in keyrings_map:
-                        keyrings_map[path].append(checksum_hash)
-                    else:
-                        keyrings_map[path] = [checksum_hash]
+                    keyrings_map[path] = add_to_list(keyrings_map.get(path), checksum_hash)
                     continue
 
                 if get_ima_buf:
-                    if path in ima_buf_map:
-                        ima_buf_map[path].append(checksum_hash)
-                    else:
-                        ima_buf_map[path] = [checksum_hash]
+                    ima_buf_map[path] = add_to_list(ima_buf_map.get(path), checksum_hash)
```

Comment:
do still need this and the rest of the changes?

Reply:
I know what you mean ... I would keep it because it makes repetitive code a lot shorter.

Reply:
To avoid the ugly repetition this can be used instead: `ima_buf_map.setdefault(path, []).append(checksum_hash)`

Reply:
Ha! Fixed.